src/datum_submitblock.c

```diff
@@ -55,10 +55,8 @@ void preciousblock(CURL *curl, char *blockhash) {
 	char userpass[1024];
```

Comment:
(same as above, and also for `datum_submitblock_doit`)

Reply:
as above